test/formatters/msbuildFormatterTests.ts

```diff
@@ -16,7 +16,7 @@
 
 import * as ts from "typescript";
 
-import {IFormatter, RuleFailure, TestUtils} from "../lint";
+import {IFormatter, RuleFailure, RuleLevel, TestUtils} from "../lint";
```

Comment:
done. Is there a way to indicate it's a breaking change for documentation purposes? Or should I just note it in the CHANGELOG.md ?